install.sh

```diff
@@ -844,16 +844,18 @@ EOABORT
   )"
 elif [[ -n "${HOMEBREW_ON_LINUX-}" ]]
 then
-  USABLE_GIT="$(find_tool git)"
-  if [[ -z "${USABLE_GIT}" ]]
+  suitable_git="$(find_tool git)"
+  if [[ -z "${suitable_git}" ]]
   then
     abort "$(
       cat <<EOABORT
 The version of Git that was found does not satisfy requirements for Homebrew.
 Please install Git ${REQUIRED_GIT_VERSION} or newer and add it to your PATH.
 EOABORT
     )"
-  elif [[ "${USABLE_GIT}" != /usr/bin/git ]]
+  fi
+  USABLE_GIT="${suitable_git}"
+  if [[ "${USABLE_GIT}" != /usr/bin/git ]]
```

Comment:
Is there a reason why you explicitly use `/usr/bin/git`?

Reply:
Don't think thats my code maybe @gromgit knows

Reply:
Oh wait yea I think its just about telling the user hey we found this git in path and going to use that.
But if we do use /usr/bin/git we just silently do that and assume the user does not care because its pretty standard.